### PR TITLE
Use sizeActual instead for file storage usage

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

- Use file actual size after encryption/compression for usage calculation instead of using sizeOriginal

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

**Review by shimonewman — APPROVED** [review]

File: tests/e2e/General/UsageTest.php [hunk 1]
```diff
@@ -328,15 +328,16 @@ public function testStorageStats(array $data): array
         $this->assertEquals(1, count($response['body']['requests']));
         $this->assertEquals($requestsTotal, $response['body']['requests'][array_key_last($response['body']['requests'])]['value']);
         $this->validateDates($response['body']['requests']);
-        $this->assertEquals($storageTotal, $response['body']['filesStorageTotal']);
+        // CANNOT check as now we use sizeActual which is not available in response
+        // $this->assertEquals($storageTotal, $response['body']['filesStorageTotal']);
```
Comment (stnguyen90) [inline tests/e2e/General/UsageTest.php:332]: Can we maybe check that it's increased?

Comment (lohanidamodar) [reply tests/e2e/General/UsageTest.php:332]: I think it should decrease, with compression?

Comment (stnguyen90) [reply tests/e2e/General/UsageTest.php]: I meant that if `storageTotal` before upload is N, then after upload, `storageTotal` should be bigger than N.

File: tests/e2e/General/UsageTest.php [hunk 2]
```diff
@@ -328,15 +328,15 @@ public function testStorageStats(array $data): array
         $this->assertEquals(1, count($response['body']['requests']));
         $this->assertEquals($requestsTotal, $response['body']['requests'][array_key_last($response['body']['requests'])]['value']);
         $this->validateDates($response['body']['requests']);
-        $this->assertEquals($storageTotal, $response['body']['filesStorageTotal']);
+        $this->assertGreaterThan($storageTotal, $response['body']['filesStorageTotal']);
```
Comment (stnguyen90) [inline tests/e2e/General/UsageTest.php:331]: This can be confusing reading the test 😬

Comment (lohanidamodar) [reply tests/e2e/General/UsageTest.php:331]: Well what do you suggest?

**Comment by lohanidamodar** [comment]
Closing instead for https://github.com/appwrite/appwrite/pull/9064

**Pull request closed without merging**